### ISSUE TITLE
convertTime() function breaks on non-English locale

### ISSUE THREAD

**Issue by szpajder** [ytti/oxidized-web#194]
I have just encountered https://github.com/ytti/oxidized-web/issues/140 which is still present in 0.13.1. The root cause is:

```Javascript
var timeZone = date.toString().match(/\(.*\)/)[0].match(/[A-Z]/g).join('');
```

When English locale are used, `date.toString()` evaluates to `"Fri Jun 14 2019 11:03:59 GMT+0200 (Central European Summer Time)"`, so the result of the above code is "CEST", as expected. But for example in Polish locale `date.toString()` yields `"Fri Jun 14 2019 11:03:59 GMT+0200 (Środkowoeuropejski czas letni)"`. `match(/[A-Z]/g)` returns NULL on this, because Ś character is not between A and Z. Taking `join('')` from NULL obviously results in an error. As a result, the Reload button does not get registered and it does not work. I guess the author of https://github.com/ytti/oxidized-web/issues/140 encountered the same problem, because, according to his pastebin, he used French locale on his system.

I changed the above to:

```Javascript
var timeZone = date.toString().match(/\+[0-9]+/)[0];
```

Now timestamps in the GUI are presented as "2019-06-14 11:03:59 +0200" instead of "2019-06-14 11:03:59 CEST". Maybe it's less pretty, but at least it works regardless of language settings.

**Comment by chmutoff** [comment]
Hi. I have exactly the same issue. Is there any fix for it?

**Comment by arddg** [comment]
@szpajder, thank you very much! This solution fixed my Reload button!

**Comment by wk** [comment]
Should be closed by #210, thanks for troubleshooting this!
